communities/src/lib.rs

```diff
@@ -456,6 +466,8 @@ impl<T: Config> Pallet<T> {
 		<CommunityMetadata<T>>::remove(cid);
 
 		<NominalIncome<T>>::remove(cid);
+
+		<encointer_balances::Pallet<T>>::purge_balances(cid).ok();
 	}
```

Comment:
Haha, I guess this is a hack to remove the unused result warning? :)

Just change the return type as advised above, and then you can remove that.